apps/email/test/config.js

```diff
@@ -91,17 +93,16 @@
 
     if (ids) {
       req(ids, function() {
-        if (callback)
+        if (callback) {
           callback.apply(null, Array.slice(arguments));
-        if (done)
-          done();
+        }
+        done && done();
```

Comment:
I would prefer to just use the `if (done) {...` form as it matches the rest of the file/usual pattern we have used for this sort of thing.